ref/feature/ldapRegistry/examples.adoc

```diff
@@ -92,9 +93,11 @@ The following example shows entity types for users and groups, with custom searc
 
 === Federation of LDAP registries
 
-Although federation of LDAP registries occurs by default, you can adjust the configuration by enabling the link:/docs/ref/feature/#federatedRegistry-1.0.html[Federated User Registry] feature and configuring the `federatedRepository` element.
+LDAP registries are federated by default.
+If you configure more than one LDAP registry in your `server.xml` file, then the registries are automatically federated into a common registry.
+You can adjust the configuration by enabling the link:/docs/ref/feature/#federatedRegistry-1.0.html[Federated User Registry] feature and configuring the `federatedRepository` element. If you configure the LDAP user registry feature version 3.0 or later, the Federated User Registry feature is enabled by default.
```

Comment:
I think "LDAP user registry feature" may need to be "LDAP User Registry feature"